clients/go/cipher.go

```diff
@@ -67,34 +47,34 @@ func NewPlainCipher() PlainCipher {
 	return PlainCipher{}
 }
 
-func (c PlainCipher) Kind() uint64 {
-	return Plain
+func (c PlainCipher) Kind() types.CallFormat {
+	return types.CallFormatPlain
```

Comment:
It feels like call format is cipher specific. So maybe this change is not the best, as if I want to add another cipher, I need to update the client-sdk library first.

Reply:
But to add a new cipher you need to update the SDK anyway?

Reply:
Yes, agreed, both should be in the SDK.